Sources/Overload/OvUI/src/OvUI/Core/UIManager.cpp

```diff
@@ -254,11 +262,60 @@ void OvUI::Core::UIManager::EnableDocking(bool p_value)
 		ImGui::GetIO().ConfigFlags &= ~ImGuiConfigFlags_DockingEnable;
 }
 
+void OvUI::Core::UIManager::ResetToDefaultLayout() const
+{
+	ImGui::LoadIniSettingsFromDisk(m_defaultLayout.string().c_str());
+}
+
 void OvUI::Core::UIManager::ResetLayout(const std::string& p_config) const
 {
     ImGui::LoadIniSettingsFromDisk(p_config.c_str());
 }
 
+void OvUI::Core::UIManager::LoadLayout(const std::string& p_fileName)
+{
+	ImGui::LoadIniSettingsFromDisk(p_fileName.c_str());
+}
+
+void OvUI::Core::UIManager::SaveLayout(const std::filesystem::path& p_filePath)
+{
+	SetEditorLayoutSaveFilename(p_filePath);
+
+	ImGui::SaveIniSettingsToDisk(m_layoutSaveFilename.string().c_str());
+}
+
+void OvUI::Core::UIManager::SaveCurrentLayout()
+{
+	if (!std::filesystem::exists(m_layoutSaveFilename))
+	{
+		m_layoutSaveFilename = (m_layoutsPath / "layout.ini").string();
```

Comment:
`.string()` here generate string, that is then used to generate another instance of `std::filesystem::path`.
```suggestion
		m_layoutSaveFilename = m_layoutsPath / "layout.ini";
```